committee/committee.go

```diff
@@ -286,6 +302,8 @@ func (ec *committee) storeInBatch(
 	results map[uint64]*types.ElectionResult,
 	errs map[uint64]error,
 ) error {
+	ec.mutex.Lock()
```

Comment:
Let's move this mutex usage to L264 in Sync method to be clear, and in case storeInBatch may be called by others